MekHQ/src/mekhq/campaign/parts/MissingMekLocation.java

```diff
@@ -265,7 +266,19 @@ public String checkFixable() {
                         }
                     }
                 }
+            } else if (slot.getType() == CriticalSlot.TYPE_EQUIPMENT) {
+                if ((slot.getMount() != null) && !slot.getMount().isDestroyed()) {
+                    EquipmentType equipmentType = slot.getMount().getType();
+                    if (equipmentType.hasFlag(MiscType.F_NULLSIG)) {
+                        return "Null-Signature System must be salvaged or scrapped first. It can then be re-installed.";
```

Comment:
Not sure that the english here parses that great. You can't re-install a system if you scrap it.

Reply:
I agree, but wanted to keep it consistent with the other messages. We could rewrite them all or leave it like this.

Reply:
Let's get rid of the re-installed bit. It's pretty redundant.

Reply:
Pushed that up, with it being removed from all of the messages with it.